### ISSUE TITLE
No errors when `"import/extensions": ["error", "ignorePackages"]`

### ISSUE THREAD

**Issue by saschanaz** [benmosher/eslint-plugin-import#1518]
It doesn't show any errors , I had to do the following instead to get the expected behavior:

```js
"import/extensions": ["error", "always", {
  "ignorePackages": true
}]
```

It seems `src/rules/extensions.js` allows but doesn't really expect `defaultConfig` to be `"ignorePackages"`.

**Comment by ljharb** [comment]
I’m not sure what the issue is here - the defaults can’t be changed without a semver-major bump.

**Comment by saschanaz** [comment]
Shouldn't `["error", "ignorePackages"]` error for extensionless non-package modules imports? Currently it never errors.

**Comment by ljharb** [comment]
Yes, it should. Can you provide code where it’s not?

**Comment by saschanaz** [comment]
AFK but I don't see any 'invalid' test in https://github.com/benmosher/eslint-plugin-import/blob/master/tests/src/rules/extensions.js

**Comment by ljharb** [comment]
True, but I’d expect it to behave the same as https://github.com/benmosher/eslint-plugin-import/blob/112a0bf442e52b25cd7029a9905df2508e191ac1/tests/src/rules/extensions.js#L298. Some example code where it doesn’t would help.
